crates/sui-graphql-rpc/tests/e2e_tests.rs

```diff
@@ -898,12 +924,412 @@ mod tests {
             .init();
         let connection_config = ConnectionConfig::ci_integration_test_cfg();
         let cluster =
-            sui_graphql_rpc::test_infra::cluster::start_cluster(connection_config, None).await;
+            sui_graphql_rpc::test_infra::cluster::start_cluster(connection_config, None, None)
+                .await;
 
         cluster
             .wait_for_checkpoint_catchup(0, Duration::from_secs(10))
             .await;
         test_health_check_impl().await;
         cluster.cleanup_resources().await
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn test_query_mutation_payload() {
+        test_query_mutation_payload_impl().await;
```

Comment:
yes, I think it can. I also do not know why separated them. Probably we added the other way first, and then we added the test cluster stuff, and went this way.

Reply:
Let's do this as a pass for all of them in a follow up PR, rather than just one.